exporter/opencensusexporter/factory.go

```diff
@@ -65,11 +64,24 @@ func (f *Factory) CreateTraceExporter(logger *zap.Logger, config configmodels.Ex
 	if err != nil {
 		return nil, nil, err
 	}
-	return f.CreateOCAgent(logger, ocac, opts)
+	oce, err := f.createOCAgentExporter(logger, ocac, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+	oexp, err := exporterhelper.NewTraceExporter(
+		"oc_trace",
+		oce.PushTraceData,
+		exporterhelper.WithSpanName("ocservice.exporter.OpenCensus.ConsumeTraceData"),
+		exporterhelper.WithRecordMetrics(true))
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return oexp, oce.stop, nil
 }
 
-// CreateOCAgent takes ocagent exporter options and create an OC exporter
-func (f *Factory) CreateOCAgent(logger *zap.Logger, ocac *Config, opts []ocagent.ExporterOption) (consumer.TraceConsumer, exporter.StopFunc, error) {
+// createOCAgentExporter takes ocagent exporter options and create an OC exporter
```

Comment:
nit: Is the term `Agent` still valid in this 'new world of OTel Svc'? I know I have a very limited understanding of OC and Otel. It seems to me that agent and collector is being more used to describe where something is running instead of the underlying mechanisms. Thank you for the explanation in advance!

Reply:
This is a legacy issue because previously we named it `ocagent` exporter (https://github.com/census-ecosystem/opencensus-go-exporter-ocagent), while it's meant to be used by both Agent and Collector. In the future when we migrate to use the opentelemetry proto, we should rename it to something like`otelsvc` exporter.